lib/index.js

```diff
@@ -244,6 +290,23 @@ exports.usage = function (definition, usage, options) {
 };
 
 
+exports.object = (opt, parsed) => {
+
+    Hoek.assert(!opt.includes('.'), `Cannot build an object at a deep path: ${opt} (contains a dot)`);
+
+    const initial = parsed.hasOwnProperty(opt) ? parsed[opt] : {};
+    const depth = (path) => path.split('.').length;
```

Comment:
This should be an `internal` method.

Reply:
I don't think that I understand why this would need to live on `internals`.  There is no styleguide rule about this that I am aware of, and in my opinion colocating this utility near where it's used is going to be a bit clearer (at least for me).